example/spec/helpers/apiUtils.js

```diff
@@ -13,21 +13,21 @@ function setDistributionApiEnvVars() {
 }
 
 /**
- * Check a record for a particular status and retry until the record gets that status
+ * Check a record for a particular set of statuses and retry until the record gets that status
  * This is to mitigate issues where a workflow completes, but there is a lag between
- * the workflow end, cloudwatch event sqs message, and dynamo update
+ * the workflow end, cloudwatch event sqs message, and database update
  *
  * @param {Object} model - model from api/models
  * @param {Object} params - params to pass to model.get
- * @param {string} status - status to wait for
+ * @param {string[] | string} status - status to wait for
```

Comment:
I explicitly avoided just refactoring all calls to this to be an array as we're going to have to refactor this shortly.